README.md

```diff
@@ -4,7 +4,7 @@ env-exampler is a script that generates a .env.example file based on the current
 
 ## Usage
 
-Run `npx env-exampler` in the directory with your .env file.
+Run `./exampler` in the directory with your .env file.
```

Comment:
You'll have to keep a local copy of the script. The npx solution is imo nicer:
- cross platform, also doesn't break the git hook on Windows machines
- all you need is npm, one short command to get it&run it
- though its probably slower

Reply:
I disagree with this, sed is standard for all posix compliant systems, also does not require an internet connection, and as above stated, does not require even npm on your system.

Reply:
> cross platform, also doesn't break the git hook on Windows machines

sed does not ship with Windows, which is still arguably the most used OS among common developers.